README.md

```diff
@@ -20,6 +20,82 @@ You can install Merlin Systems with `pip`:
 pip install merlin-systems
 ```
 
+### Installing Merlin Systems from Source
```

Comment:
Let's move this section to a CONTRIBUTING.md file. Maybe we can steal one from another Merlin repo?

Reply:
+1 that this is better in CONTRIBUTING.md along with the suggestion to install pre-commit and that docstrings are numpy flavor.  I need more hours in my days.